deploy/grafana-service.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: "v1beta1"
 kind: "Service"
 id: "monitoring-grafana"
-port: 8087
+port: 80
```

Comment:
Kube DNS service assumes port '80' to be the default port for all services. It is not identifying the port specified for the service in the Spec. This might be a bug that has already been fixed. Since every service has its own IP, the port here doesn't really matter much.